packages/tools/fluid-runner/src/logger/loggerUtils.ts

```diff
@@ -85,5 +86,12 @@ export function validateAndParseTelemetryOptions(
 		}
 	}
 
-	return { success: true, telemetryOptions: { outputFormat, defaultProps } };
+	if (eventsPerFlush !== undefined && isNaN(eventsPerFlush)) {
```

Comment:
#nit MDN says `isNaN` can be 'surprising' 😄  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN